README.md

```diff
@@ -90,6 +90,7 @@ break_before_table_rb: true
 chop_down_table: false
 chop_down_kv_table: true
 table_sep: ","
+column_table_limit: column_limit
```

Comment:
The default value of `column_table_limit` is not a reference to `column_limit `.
Consider this config file:
```yaml
column_limit: 120
```
The value of `column_table_limit` is 80, not 120.

Reply:
ok, I figured it out.
I fixed it by checking the config file and updating its value with `column_limit` .